internal/website/content/howto/pubsub/subscribe.md

```diff
@@ -555,19 +225,7 @@ You will also need to pass an acknowledgement deadline: once a message is
 received, if it is not acknowledged after the deadline elapses, then it will be
 redelivered.
 
-```go
-import (
-    "time"
-
-    "gocloud.dev/pubsub/mempubsub"
-)
-
-// ...
-
-topic := mempubsub.NewTopic()
-defer topic.Shutdown(ctx)
-subscription := mempubsub.NewSubscription(topic, 1 * time.Minute)
-```
+{{< goexample "gocloud.dev/pubsub/mempubsub.ExampleNewSubscription" >}}
```

Comment:
Question: if you typo this and `ExampleNewSubscription` does not exist, will Travis catch that?

Reply:
Good point. Added logic to the shortcode to make it do exactly that.